src/main/java/com/google/devtools/build/lib/skyframe/PrerequisiteProducer.java

```diff
@@ -335,9 +358,14 @@ public boolean evaluate(
           unloadedToolchainContexts != null ? unloadedToolchainContexts.getTargetPlatform() : null;
 
       // Get the configuration targets that trigger this rule's configurable attributes.
-      configConditions =
-          computeConfigConditions(
-              env, targetAndConfiguration, transitivePackages, platformInfo, transitiveRootCauses);
+      // May have been computed as part of checking for incompatible targets at the beginning of
```

Comment:
Under what conditions would it not have been? When toolchain resolution is disabled for a rule?

Reply:
Yes, that's the only case, added that to the comment. This `if` is completely optional, if you think duplicating the call in the common case won't matter, I can get rid of it.

Reply:
That's fine, leave it as-is.